src/main/java/hudson/plugins/git/extensions/BrowserInference.java

```diff
@@ -0,0 +1,22 @@
+package hudson.plugins.git.extensions;
+
+import hudson.plugins.git.browser.GitRepositoryBrowser;
+import hudson.scm.RepositoryBrowsers;
+import org.apache.tools.ant.ExtensionPoint;
```

Comment:
As https://wiki.jenkins.io/display/JENKINS/Defining+a+new+extension+point mention, you're using the wrong extension point class
```suggestion
import hudson.ExtensionPoint;
```